### PR TITLE
feat:talk details data

### PR DESCRIPTION
**Pull request by kateinkim**
# Description

[Trello Card](75-real-data-talk-details)

Summary of the changes: 

- Updated files that are necessary to bring real data into the talk details screen
- Added `talkDetailsProps` to render different data per schedule type
- Recurring event card will show start time to end time

# Graphics
Note: Jamon doesn't have a bio in the CMS so it's not showing

https://user-images.githubusercontent.com/53795920/212795445-085f60e2-fdff-4c9c-a7cb-23ace40fbf14.mp4

|Time on the recurring events before|After 1|After 2|
|-|-|-|
|CleanShot_2023-01-16_at_10 50 44@2x|CleanShot 2023-01-17 at 11 05 33@2x|CleanShot 2023-01-17 at 11 05 52@2x|

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

**Comment by kateinkim** [comment]
[Real data: Talk Details](https://trello.com/c/mqkzk9IX/75-real-data-talk-details)

**Comment by robinheinze** [comment]
When I run it locally, I see the time as a number. Is that coming from webflow directly? 

Screen Shot 2023-01-18 at 11 05 18 AM

**Comment by kateinkim** [comment]
> When I run it locally, I see the time as a number for workshops (looks fine for talks). Is that coming from webflow directly?
> 
Somehow transforming the location data part got lost after I took the video in the description. 😓

File: app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx [hunk 1]
```diff
@@ -63,40 +148,36 @@ export const TalkDetailsScreen: FC<StackScreenProps<AppStackParamList, "TalkDeta
           </View>
           <View style={$containerSpacing}>
             <Image
-              source={isWorkshop ? workshopCurve : talkCurve}
-              style={isWorkshop ? $workshopCurve : $talkCurve}
+              source={variant === "workshop" ? workshopCurve : talkCurve}
```
Comment (robinheinze) [inline app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx:151]: Nit: Maybe eliminate some duplication by re-purposing the `isWorkshop` variable as `const isWorkshop = variant === "workshop"`?

**Review by robinheinze — APPROVED** [review]
Looks fantastic! Just one minor nit.

**Pull request merged**